### PR TITLE
Update helm blob location

### PR DESCRIPTION
**Pull request by bkochendorfer**
The URL for obtaining the Helm blob has changed.

https://helm.sh/blog/get-helm-sh/

https://github.com/mozilla-iam/iam-infra/pull/299

### CODE REVIEW

**Comment by bkochendorfer** [comment]
Updated but I don't think we build this anywhere that I can find

**Review by gene1wood — CHANGES_REQUESTED** [review]
It looks like that's not where to get helm from. The helm.sh site says to get it from GitHub

https://github.com/helm/helm/releases/

The URL in the PR doesn't work.

**Comment by bkochendorfer** [comment]
oh shoot, I tested the curl but didn't realize it was just text.

File: buildspec.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ phases:
       - curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
       - echo "deb http://apt.kubernetes.io/ kubernetes-xenial main" | tee -a /etc/apt/sources.list.d/kubernetes.list
       - apt update && apt install -y kubectl
-      - curl -O https://storage.googleapis.com/kubernetes-helm/helm-v2.11.0-linux-amd64.tar.gz
+      - curl -O https://get.helm.sh/kubernetes-helm/helm-v2.11.0-linux-amd64.tar.gz
```
Comment (bkochendorfer) [inline buildspec.yml:14]:
```suggestion
      - curl -O https://get.helm.sh/helm-v2.11.0-linux-amd64.tar.gz
```

**Comment by bkochendorfer** [comment]
```
$ curl -O https://get.helm.sh/helm-v2.11.0-linux-amd64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 18.2M  100 18.2M    0     0  14.0M      0  0:00:01  0:00:01 --:--:-- 14.1M
bkochendorfer@mozilla:/tmp\ 
$ file helm-v2.11.0-linux-amd64.tar.gz 
helm-v2.11.0-linux-amd64.tar.gz: gzip compressed data, last modified: Tue Sep 25 18:17:08 2018, from Unix, original size modulo 2^32 63784960
bkochendorfer@mozilla:/tmp\ 
$ tar xvf helm-v2.11.0-linux-amd64.tar.gz 
linux-amd64/
```

**Comment by bkochendorfer** [comment]
@gene1wood This seems to work

**Review by gene1wood — APPROVED** [review]

**Pull request merged**